cli/src/config.rs

```diff
@@ -94,6 +95,15 @@ pub struct AuthConfig {
     pub refresh_token: String,
 }
 
+// ReleaseInfo stores information about a release
+#[derive(Serialize, Deserialize, Debug, PartialEq, Eq, Clone)]
+pub struct ReleaseInfo {
+    pub version: String,
+    pub url: String,
+    pub published_at: String,
+    pub checked_for_update_at: String,
```

Comment:
What is the purpose of storing all these informations into the config file? from what I understand, we only need the `checked_for_update_at` value.

Reply:
Btw I think `checked_for_update_at` is not a good name. I suggest change it to something like `last_checked_at` or `last_update_checked_at`.